### ISSUE TITLE
NTR: Mycobacterium present in CSF microscopy (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8060]
New term request Mycobacterium present in CSF microscopy (samant_101122010105): Cerebrospinal fluid  UMLS: 0855371

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000544
- label: Positive mycobacterium CSF microscopy
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: PMID:28105125
- def: The demonstration of mycobacterium by microscopy of the cerebrospinal fluid (CSF).
- comment: Ziehl-Neelsen (Z-N) staining of cerebrospinal fluid (CSF) for acid‑fast bacilli (AFB) is the cornerstone of the laboratory diagnosis of tuberculous meningitis (TBM). However, the sensitivity of conventional Z‑N staining for the detection of AFB in CSF specimens is suboptimal.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8060
- synonyms:
